autoreject/autoreject.py

```diff
@@ -566,7 +566,8 @@ def get_reject_log(self, epochs, threshes=None, picks=None):
             bad_sensor_counts, ch_type=ch_type, picks=this_picks)
 
         reject_log = RejectLog(labels=labels, bad_epochs=bad_epochs,
-                               ch_names=epochs.ch_names)
+                               ch_names=epochs.ch_names,
+                               bad_sensor_counts=bad_sensor_counts)
```

Comment:
this can be obtained from the labels no? If so I would not add this param to constructor but no objection to add some in the API of RejectLog to get this info